tutorials/cloudsdk-js-odata-batch-changeset/cloudsdk-js-odata-batch-changeset.md

```diff
@@ -10,7 +10,7 @@ primary_tag: products>sap-s-4hana-cloud-sdk
 ## Prerequisites
  - Have `Node.js` and `npm` [installed on your machine](s4sdkjs-prerequisites).
  - Have access to an SAP S/4HANA Cloud system or the [SAP API Business Hub Sandbox](https://api.sap.com/getting-started), or use the [Business Partner Mock Service](https://sap.github.io/cloud-s4-sdk-book/pages/mock-odata.html).
- - Learn how to use the SAP Cloud SDK's Virtual Data Model by going through [these tutorials](https://developers.sap.com/group.cloudsdk-js-vdm.html).
+ - Recommended to explored the more basic CRUD operations by going through [these tutorials](https://developers.sap.com/group.cloudsdk-js-vdm.html), but not required.
```

Comment:
I think "explored" is meant to be "explore"? Also, links to groups and missions and tutorials are abbreviated this way:

- tutorials (cloudsdk-mytutorial)
- groups  (group.cloudsdk-mygroup)
- missions (mission.clodsdk-mymission)

Can you update?

Reply:
I went over the tutorials and looked for too long links. I hope I did not miss any of those.